src/cdk/overlay/overlay.spec.ts

```diff
@@ -532,9 +532,14 @@ class OverlayTestModule { }
 class OverlayContainerThemingTestModule { }
 
 class FakePositionStrategy implements PositionStrategy {
-  apply(element: Element): Promise<null> {
-    element.classList.add('fake-positioned');
-    return Promise.resolve(null);
+  element: HTMLElement;
+
+  apply(): void {
+    this.element.classList.add('fake-positioned');
```

Comment:
I don't follow what you mean; the existing test for this still applies- is there something that isn't covered?